blosc-sys/build.rs

```diff
@@ -15,6 +15,18 @@ fn main() {
     } else {
         None
     };
+    #[cfg(unix)]
+    if bindings.is_none() {
+        if let Ok(pc) = pkg_config::probe_library("blosc") {
```

Comment:
If you're going to go to the trouble of using pkg-config for this case when the header isn't in the default include path, why don't just always use pkg-config?

Reply:
Good idea! I'll change this up! The idea was to have this as a fallback only for supported platforms.